src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/SynchronizeTest.kt

```diff
@@ -95,4 +95,24 @@ class SynchronizeTest: AbstractTest() {
 
         }
     }
+
+    /**
+     * Tests /v1/synchronizeDeletions -endpoint
+     * with mock Forecast data from past 30
+     */
+    @Test
+    fun testDeletedSynchronization() {
+        createTestBuilder().use { testBuilder ->
+            testBuilder.manager.synchronization.synchronizeEntries()
+
+            setScenario(
+                scenario = TIMES_SCENARIO,
+                state = "delete"
+            )
+
+            testBuilder.manager.synchronization.synchronizeDeletedEntries()
+            val timeBankTimeEntries = testBuilder.manager.timeEntries.getTimeEntries()
```

Comment:
Am I missing something or isn't this test missing assertions?

Reply:
Yes it is. My bad i must have only dreamed about adding them in.